Dockerfile.kernel

```diff
@@ -1,15 +1,21 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-FROM jupyter/pyspark-notebook:0017b56d93c9
+FROM jupyter/all-spark-notebook:0017b56d93c9
 
 RUN pip install jupyter_kernel_gateway==0.5.0
 # install ipywidgets 5.x
 RUN pip install 'ipywidgets==5.1.3'
 
 # install Declarative Widgets python package
 # don't bother activating the extension, not needed outside notebook
-RUN pip install jupyter_declarativewidgets
+RUN pip install jupyter_declarativewidgets==0.5.2
+
+# also install above packages for python2
+USER root
+RUN /opt/conda/envs/python2/bin/pip install 'ipywidgets==5.1.3'
+RUN /opt/conda/envs/python2/bin/pip install jupyter_declarativewidgets==0.5.2
+USER jovyan
```

Comment:
No. Something's wrong if we can't run those as jovyan. I bet we need to activate the conda env properly. Like `bash -c "source activate python2 && <all the rest of the command lumped in>"`